ert/storage/_storage.py

```diff
@@ -396,52 +396,42 @@ def _add_numerical_data(
         raise ert.exceptions.StorageError(meta_response.text)
 
 
+# the local variable dataframe is not read by pylint as pandas.Dataframe,
+# but due to chunking a pandas.TextFileReader
+# https://stackoverflow.com/questions/41844485/why-the-object-which-i-read-a-csv-file-using-pandas-from-is-textfilereader-obj
 def _response2records(
     response_content: bytes, metadata: _NumericalMetaData
 ) -> ert.data.RecordCollection:
-
+    assert metadata
     record_type = metadata.record_type
-
-    # the local variable dataframe is not read by pylint as pandas.Dataframe,
-    # but due to chunking a pandas.TextFileReader
-    # https://stackoverflow.com/questions/41844485/why-the-object-which-i-read-a-csv-file-using-pandas-from-is-textfilereader-obj
-
-    records: List[ert.data.Record]
-    if record_type == ert.data.RecordType.LIST_FLOAT:
-        dataframe = pd.read_csv(
-            io.BytesIO(response_content), index_col=0, float_precision="round_trip"
+    if record_type not in ert.data.RecordType.__members__:
```

Comment:
I am not sure it is ok to use protected elements like `__members__` outside the `RecordType` class :shrug:

Reply:
`if record_type not in set(item.value for item in ert.data.RecordType)`
would that be better? or should it stay how it was?

Reply:
if you want the check like then 

```python
if record_type not in set(item.value for item in ert.data.RecordType)
```
is fine by me. 

But I am not really sure this check is required at the moment, so it can be removed in my opinion